content/platforms/kubernetes/managing-images.md

```diff
@@ -0,0 +1,295 @@
+Red Hat, and other public registries for distribution. Your organization may
+require these images be copied to other registries used by your Kubernetes
+clusters.
+
+### Creating a private repository
+
+A private registry can be setup via:
+
+ * on-premise via [Docker registry](https://docs.docker.com/registry/deploying/),
+   RedHat Quay, or other providers,
+ * via a cloud provider (e.g., Azure Container Registry, Google Container Registry, etc.).
+
+Once you have setup a repository, you will have:
+
+ * a domain name,
+ * an optional port,
+ * an optional registry path suffix,
+
+This information identifies your registry and must be used to references images you
+push to your private registry. For example, a Google Container Registry
+will start with `gcr.io/{project-id}` where '{project-id}' is the cloud project
+identifier.
+
+### Pushing Redis Enterprise images to a private repository
+
+A Kubernetes deployment uses different images for various purposes:
+
+ * Redis Enterprise
+ * bootstrapping a Redis Enterprise cluster node
+ * the Service Rigger
+ * the Redis Enterprise operator
+
+Once you have setup a private repository, you will need to push these images
+to your private repository. The process to do so is:
+
+ 1. Pull the various images locally for Redis Enterprise and the operator.
+ 2. Tag the local images with the repository prefixed onto the name.
+ 3. Push the new tagged images name with the repository prefix.
+
+For Redis Enterprise and the operator, this process can be accomplished by:
+
+```
+PRIVATE_REPO=...your repo...
+RS_VERSION=6.0.8-28
+OPERATOR_VERSION=6.0.8-1
+docker pull redislabs/redis:${RS_VERSION}
+docker pull redislabs/operator:${OPERATOR_VERSION}
+docker pull redislabs/k8s-controller:${OPERATOR_VERSION}
+docker tag redislabs/redis:${RS_VERSION} ${PRIVATE_REPO}/redislabs/redis:${RS_VERSION}
+docker tag redislabs/operator:${OPERATOR_VERSION} ${PRIVATE_REPO}/redislabs/operator:${OPERATOR_VERSION}
+docker tag redislabs/k8s-controller:${OPERATOR_VERSION} ${PRIVATE_REPO}/redislabs/k8s-controller:${OPERATOR_VERSION}
+docker push ${PRIVATE_REPO}/redislabs/redis:${RS_VERSION}
+docker push ${PRIVATE_REPO}/redislabs/operator:${OPERATOR_VERSION}
+docker push ${PRIVATE_REPO}/redislabs/k8s-controller:${OPERATOR_VERSION}
+```
+
+## Using a private repository
+
+After pushing your desired version to your private repository, you just need to
+configure your deployments to use the private repository. There are two
+different deployments to consider:
+
+ 1. The Redis Enterprise operator.
+ 2. The Redis Enterprise pods and Service Rigger created by the operator.
+
+For (1), the operator container image is controlled directly by the deployment
+bundle. Whereas, for (2), the Redis Enterprise cluster pod (RS and bootstrapper) and Service Rigger
+images are controlled in the Redis Enterprise Custom Resource.
+
+Also, depending on your Kubernetes platform, your private repository may
```

Comment:
Again, I suggest choosing "registry" or "repository" but not both.

Reply:
clarified use of container registry vs repository throughout the doc